Cargo.toml

```diff
@@ -65,6 +69,9 @@ tree-sitter-swift = "0.3.6"
 tree-sitter-toml = "0.20.0"
 tree-sitter-typescript = "0.20.2"
 
+[patch.crates-io]
+tree-sitter = { git = "https://github.com/helixbass/tree-sitter", rev = "57e98fb0" }
```

Comment:
Read about how to do this "force dependencies to use this version"

The thing that I guess I wasn't anticipating is that I also had to do this in any "outer" crates (`tree-sitter-lint` and its project-local crates) even though they don't have their own explicit dependency on `tree-sitter` (they use `tree-sitter-grep`'s re-export)?